roboc/carte.py

```diff
@@ -5,6 +5,7 @@
 
 """Ce module contient la classe Carte."""
 
+
```

Comment:
`SELECT * FROM A INNER JOIN  B ON A.id = B.a_id where c is not null`

`result = A.join(B,  on= A.id == B.a_id, how= “inner”).where(col(c).isNotNull())`